src/endpoints/create.rs

```diff
@@ -1,51 +1,59 @@
+use crate::args::Args;
+use crate::error_handling::AppError;
 use crate::pasta::PastaFile;
 use crate::util::animalnumbers::to_animal_names;
 use crate::util::db::insert;
 use crate::util::hashids::to_hashids;
 use crate::util::misc::{encrypt, encrypt_file, is_valid_url};
 use crate::{AppState, Pasta, ARGS};
-use actix_multipart::Multipart;
-use actix_web::error::ErrorBadRequest;
-use actix_web::{get, web, Error, HttpResponse, Responder};
 use askama::Template;
+use axum::extract::{Multipart, Path, State};
+use axum::http::Response;
+use axum::response::IntoResponse;
+use axum::routing::{get, post};
+use axum::Router;
 use bytesize::ByteSize;
 use futures::TryStreamExt;
 use log::warn;
-use rand::Rng;
-use std::io::Write;
+use std::sync::LazyLock;
 use std::time::{SystemTime, UNIX_EPOCH};
+use tokio::io::AsyncWriteExt;
 
 #[derive(Template)]
 #[template(path = "index.html")]
 struct IndexTemplate<'a> {
-    args: &'a ARGS,
+    args: &'a LazyLock<Args>,
```

Comment:
Just in case you would like even more work to do :stuck_out_tongue: 

I would love a future PR that overhauls the args system. In my opinion args should be parsed in main and then passed on to everything that needs them (maybe via the state struct). Having them global but them borrowing again in the templates is kinda strange if you ask me.

Reply:
I agree. We should pass this as state around and then just access it in the handler and below. Generally globals are quite bad for testing etc.